s2s_client.go

```diff
@@ -0,0 +1,150 @@
+// Server to server (S2S) API from Zignsec
+
+package zignsec
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+const (
+	// S2SURL is the production endpoint
+	S2SURL = "https://api.zignsec.com/v2/BankIDSE"
+	// S2SURLTest is the test endpoint
+	S2SURLTest = "https://test.zignsec.com/v2/BankIDSE"
+	// Timeout3 is how long BankID will wait before cancelling a request
+	Timeout3 = time.Duration(3 * time.Minute)
+)
+
+// S2SClient is a Zignsec server to server client
+type S2SClient struct {
+	baseURL string
+	key     string
+	timeout time.Duration
+}
+
+// NewS2SClient create a new client.
+// Key is the Zignsec Authorization.
+// Timeout is only used by Authenticate() and Sign().
+func NewS2SClient(baseURL string, key string, timeout time.Duration) *S2SClient {
+	result := S2SClient{baseURL: baseURL, key: key, timeout: timeout}
+	return &result
+}
+
+// Authenticate request that does both Init() and Collect().
+func (c *S2SClient) Authenticate(nr string) (*CollectResponse, error) {
```

Comment:
This should also be removed.

This library is only for talking to the API, not provide any extra logic.